### ISSUE TITLE
Problem with docker-compose up on Windows

### ISSUE THREAD

**Issue by JacSta** [CodeForPoznan/volontulo#670]
After executing `docker-compose up` there is a problem with backend container .

Stacktrace:
```
docker-compose up
Starting volontulo_frontend_1 ...
Starting volontulo_db_1 ...
Starting volontulo_frontend_1
Starting volontulo_db_1 ... done
Starting volontulo_backend_1 ...
Starting volontulo_backend_1 ... done
Attaching to volontulo_db_1, volontulo_frontend_1, volontulo_backend_1
db_1        | LOG:  database system was shut down at 2017-09-27 16:15:39 UTC
db_1        | LOG:  MultiXact member wraparound protections are now enabled
db_1        | LOG:  database system is ready to accept connections
db_1        | LOG:  autovacuum launcher started
: invalid optioncker-entrypoint.sh: line 2: set: -
backend_1   | set: usage: set [-abefhkmnptuvxBCHP] [-o option-name] [--] [arg ...]
backend_1   | docker-entrypoint.sh: line 3: $'\r': command not found
backend_1   | docker-entrypoint.sh: line 25: syntax error: unexpected end of file
frontend_1  | Your global Angular CLI version (1.4.3) is greater than your local
frontend_1  | version (1.0.0). The local Angular CLI version is used.
frontend_1  |
frontend_1  | To disable this warning use "ng set --global warnings.versionMismatch=false".
volontulo_backend_1 exited with code 2
frontend_1  | ** NG Live Development Server is running on http://0.0.0.0:4200 **
frontend_1  |  10% building modules 6/10 modules 4 active .../bootstrap/dist/css/bootstrap.min.css(node:1) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection
id: 1): Error: Exited with code 3
frontend_1  | (node:1) DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process wi
th a non-zero exit code.
Gracefully stopping... (press Ctrl+C again to force)
```

I think there is an issue with docker-entrypoint.sh bash script file.

**Comment by magul** [comment]
fixed in #716

**Comment by Simon323** [comment]
Pull request: https://github.com/CodeForPoznan/volontulo/pull/716
Solved
